app/src/main/res/layout/item_talk_topic.xml

```diff
@@ -13,9 +13,11 @@
         android:id="@+id/topicReadDot"
         android:layout_width="8dp"
         android:layout_height="8dp"
+        android:layout_marginEnd="12dp"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/topic_title_text"
```

Comment:
Please update it to `@id/topicTitleText`

Reply:
whoops